src/gsy_e/models/strategy/trading_strategy_base.py

```diff
@@ -94,3 +94,8 @@ def event_market_cycle(self):
     def state(self) -> "StateInterface":
         """Get the state class of the strategy. Needs to be implemented by all strategies"""
         raise NotImplementedError
+
+    @property
+    def trades(self):
```

Comment:
Not saying to remove this, because I believe that this will be very useful in the long run, but I did not spot this being used in the actual strategy, right ?

Reply:
I had to learn the hard way, that this is needed in the [endpoint_buffer.py](https://github.com/gridsingularity/gsy-e/blob/8a8b0a4e5e651db8eb2b1b55665b68dc2afaa145/src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py#L322)

Reply:
Thanks for pointing this out, and too bad to hear that! I am currently wondering whether it makes sense to add it as an abtractproperty on the AreaBehaviorBase class and fix the current BaseStrategy class to handle this, however since we plan to slowly migrate away from these classes please leave it as it is, no need to do this now.